htdocs/hrm/class/evaluation.class.php

```diff
@@ -186,7 +186,7 @@ public function __construct(DoliDB $db)
 		}
 
 		if (!$user->hasRight('hrm', 'evaluation', 'readall')) {
-			$this->fields['fk_user']['type'] .= ':rowid IN('.$this->db->sanitize(implode(", ", $user->getAllChildIds(1))).')';
+			$this->fields['fk_user']['type'] .= '(:t.rowid:in:'.implode(",", $user->getAllChildIds(1));
```

Comment:
Do you mean
`$this->fields['fk_user']['type'] .= '(t.rowid:in:'.implode(",", $user->getAllChildIds(1));`
instead of
`$this->fields['fk_user']['type'] .= '(:t.rowid:in:'.implode(",", $user->getAllChildIds(1));`

Reply:
if I remove the “:” character before the t.rowid my filter looks like this : 

_'in:109,121,145,24,148,204,107,209,70,141,168,152,44,200,68,69,138,86,205,118,129,74,122,14'_

to work it should look like this: 

_'t.rowid:in:109,121,145,24,148,204,107,209,70,141,168,152,44,200,68,69,138,86,205,118,129,74,122,14'_

keeping this syntax seems fine to me,
do you have any other suggestions that I can't see?